src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -117,7 +117,8 @@ export function loadStateFromAuthoring(state, authoring, progress={}) {
         numTrials = authoredChallenge.numTrials || (trials ? trials.length : 1),
         trialOrder = createTrialOrder(trial, trials, state.trialOrder, authoredChallenge.randomizeTrials),
         drakes = processAuthoredDrakes(authoredChallenge, trialOrder[trial], template, trial),
-        gametes = processAuthoredGametes(authoredChallenge, drakes, state);
+        gametes = processAuthoredGametes(authoredChallenge, drakes, state),
+        zoomUrl = authoredChallenge.zoomUrl || null;
```

Comment:
Totally true, fixed it.

I don't know at this point whether it would also be worth removing props from the state tree if they don't exist, instead of leaving them cluttered there, but not sure whether it's worth it.